src/taxonomy/TaxonomyListPage.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {
+  Button,
+  CardView,
+  Container,
+  DataTable,
+  FormControl,
+  Spinner,
+} from '@edx/paragon';
+import { Add } from '@edx/paragon/icons';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import Header from '../header';
+import SubHeader from '../generic/sub-header/SubHeader';
+import messages from './messages';
+import TaxonomyCard from './TaxonomyCard';
+import { useTaxonomyListDataResponse, useIsTaxonomyListDataLoaded } from './api/hooks/selectors.ts';
+
+const TaxonomyListPage = ({ intl }) => {
+  const orgDefaultValue = intl.formatMessage(messages.orgInputSelectDefaultValue);
+
+  const useTaxonomyListData = () => {
+    const taxonomyListData = useTaxonomyListDataResponse();
+    const isLoaded = useIsTaxonomyListDataLoaded();
+    return { taxonomyListData, isLoaded };
+  };
+
+  const { taxonomyListData, isLoaded } = useTaxonomyListData();
+
+  const getHeaderButtons = () => [
+    (
+      <Button
+        variant="link"
+        className="text-dark-900"
+      >
+        {intl.formatMessage(messages.downloadTemplateButtonLabel)}
```

Comment:
Download template isn't required for this story - it comes later in https://github.com/openedx/modular-learning/issues/114 . But if you can easily implement it now, feel free to do so.